python/pyspark/serializers.py

```diff
@@ -96,7 +96,7 @@ def load_stream(self, stream):
         raise NotImplementedError
 
     def _load_stream_without_unbatching(self, stream):
```

Comment:
Even though this is internal it might make sense to have a docstring for this since were changing its behaviour.